src/Workspaces/CoreTestUtilities/Remote/InProcRemoteHostClientProvider.cs

```diff
@@ -17,6 +17,7 @@
 
 namespace Microsoft.CodeAnalysis.Remote.Testing
 {
+#pragma warning disable CA1416 // Validate platform compatibility
```

Comment:
>CA1416 [](http://example.com/codeflow?start=24&length=6)

I'm curious why this didn't use the SupportedOSPlatform attribute too

Reply:
it just got very viral.